prov/efa/src/rdm/rxr_env.c

```diff
@@ -120,13 +83,17 @@ void rxr_env_define_use_device_rdma()
 	int use_device_rdma;
 
 	/* Get the default value of using device's RDMA capability */
-	use_device_rdma = rxr_env_get_default_use_device_rdma();
+	use_device_rdma = rxr_get_default_use_device_rdma();
 
 	/* Specify the help info about the usage of RDMA in the device. */
 	if (!(g_device_list[0].efa_attr.device_caps & EFADV_DEVICE_ATTR_CAPS_RDMA_READ))
-		str = "EFA device on your system does not support RDMA, so this variable cannot be set to 1";
+		str = "EFA device on your system does not support RDMA,"
+			" so this variable cannot be set to 1";
 	fi_param_define(&efa_prov, "use_device_rdma", FI_PARAM_BOOL,
-			"Specifies whether to use device's RDMA functionality for one-sided and two-sided transfer. (Default: %d). %s", use_device_rdma, str);
+			"Ignored if user selects libfabric API version >=1.18."
```

Comment:
I do not think we ignore the environment variable.

Reply:
No, not anymore.  Thank you, fixed.